src/fuzz/fuzz-dns-packet.c

```diff
@@ -21,23 +21,20 @@
 
 int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
         _cleanup_(dns_packet_unrefp) DnsPacket *p = NULL;
-        int r;
 
         if (size > DNS_PACKET_SIZE_MAX)
                 return 0;
 
-        r = dns_packet_new(&p, DNS_PROTOCOL_DNS, 0, DNS_PACKET_SIZE_MAX);
-        if (r < 0)
-                return 0;
+        assert_se(dns_packet_new(&p, DNS_PROTOCOL_DNS, 0, DNS_PACKET_SIZE_MAX) >= 0);
         p->size = 0; /* by default append starts after the header, undo that */
-        dns_packet_append_blob(p, data, size, NULL);
+        assert_se(dns_packet_append_blob(p, data, size, NULL) >= 0);
         if (size < DNS_PACKET_HEADER_SIZE) {
                 /* make sure we pad the packet back up to the minimum header size */
-                assert(p->allocated >= DNS_PACKET_HEADER_SIZE);
+                assert_se(p->allocated >= DNS_PACKET_HEADER_SIZE);
                 memzero(DNS_PACKET_DATA(p) + size, DNS_PACKET_HEADER_SIZE - size);
                 p->size = DNS_PACKET_HEADER_SIZE;
         }
-        dns_packet_extract(p);
+        assert_se(dns_packet_extract(p) >= 0);
```

Comment:
As I've just written above, placing `(void)` here seems to be a better solution to please `Coverity`.